client/apollo/react/src/Form/checkbox/checkbox/CheckboxCommon.tsx

```diff
@@ -0,0 +1,44 @@
+import React from "react";
+import { Svg } from "../../../Svg/Svg";
+
+type CheckboxIcon = {
+  checkBoxIcon: string;
+};
+
+export type CheckboxProps = {
+  inputProps?: React.InputHTMLAttributes<HTMLInputElement>;
+  inputId?: string;
+  idError?: string;
+  isError?: boolean;
+  className?: string;
+  onChange?: React.ChangeEventHandler;
+} & Omit<React.InputHTMLAttributes<HTMLInputElement>, "label" | "disabled">;
+
+export type CheckboxCommonProps = CheckboxProps & CheckboxIcon;
+const CheckboxCommon = ({
+  inputProps,
+  inputId,
+  idError,
+  checkBoxIcon,
+  isError,
+  onChange,
+}: CheckboxCommonProps) => (
+  <span className="af-checkbox">
+    <input
+      {...inputProps}
+      type="checkbox"
+      aria-label="checkbox input"
+      id={inputId}
+      aria-errormessage={idError}
+      aria-invalid={isError}
+      disabled={false}
+      onChange={onChange}
+    />
+    <span className="af-checkbox__icons">
+      <Svg src={checkBoxIcon} className="af-checkbox__checked" />
+    </span>
+  </span>
+);
```

Comment:
Autant laisser les développeurs surcharger ce qu'ils veulent en définissant des valeurs par défaut.
De plus, pas besoin de définir manuellement le onChange, la destructuration des props restantes va l'ajouter automatiquement.

```suggestion
const CheckboxCommon = ({
  inputProps,
  inputId,
  idError,
  checkBoxIcon,
  isError,
}: CheckboxCommonProps) => (
  <span className="af-checkbox">
    <input
      type="checkbox"
      aria-label="checkbox input"
      id={inputId}
      aria-errormessage={idError}
      aria-invalid={isError}
      disabled={false}
      {...inputProps}
    />
    <span className="af-checkbox__icons">
      <Svg src={checkBoxIcon} className="af-checkbox__checked" />
    </span>
  </span>
);
```